Modelica_DeviceDrivers/UsersGuide.mo

```diff
@@ -83,22 +83,59 @@ This library is based on various resources (mainly within the internet) describi
 
   package ReleaseNotes "Release notes"
     extends Modelica.Icons.ReleaseNotes;
+    class Version_1_6_0 "Version 1.6.0 (October 6, 2018)"
+    extends Modelica.Icons.ReleaseNotes;
+
+      annotation (Documentation(info="<html>
+<ul>
+<li>Support for MQTT (Message Queuing Telemetry Transport protocol) client communication (<a href=\"https://github.com/modelica/Modelica_DeviceDrivers/issues/130\">#130</a>, <a href=\"https://github.com/modelica/Modelica_DeviceDrivers/issues/256\">#256</a>).
+See example <a href=\"modelica://Modelica_DeviceDrivers.Blocks.Examples.TestSerialPackager_MQTT\">TestSerialPackager_MQTT</a>.
+</li>
+<li>
+Utility function to retrieve MAC address (<a href=\"modelica://Modelica_DeviceDrivers.Utilities.Functions.getMACAddress\">getMACAddress</a>, <a href=\"https://github.com/modelica/Modelica_DeviceDrivers/issues/255\">#255</a>).
+</li>
+<li>
+Utility function to generate a UUID (<a href=\"modelica://Modelica_DeviceDrivers.Utilities.Functions.generateUUID\">generateUUID</a>, <a href=\"https://github.com/modelica/Modelica_DeviceDrivers/issues/244\">#244</a>).
+</li>
+<li>
+Number of received bytes in `UDPReceive` block are provided as outputs (<a href=\"https://github.com/modelica/Modelica_DeviceDrivers/issues/236\">#236</a>).
+</li>
+<li>
+Scalable real-time synchronization (<a href=\"https://github.com/modelica/Modelica_DeviceDrivers/issues/215\">#215</a>).
+</li>
+<li>
+Adaption of the new Modelica Association license: <b>BSD-3 clause</b>
```

Comment:
Can you call it `BSD 3-Clause` or `3-Clause BSD`, i.e., with the dash after the 3.